### PR TITLE
[DEITS] Add force option

### PR DESCRIPTION
**Pull request by innerdvations**
NOTE: I accidentally based this off of https://github.com/strapi/strapi/pull/15406 so that should be reviewed and merged before this one so I don't have to rebase 😆

### What does it do?

Adds the --force option to `strapi import`

### Why is it needed?

To allow the command to be run non-interactively

### How to test it?

run `strapi import -f ...` and the restore process should be automatically accepted without interaction
run `strapi import ...` without -f and the restore process should prompt before deleting the database

### CODE REVIEW

**Comment by Convly** [comment]
Hey, I've changed the base branch on the PR to have more visibility on the changes.
When #15406 will be merged, Github will automatically update the base branch of this PR to `features/deits` (the branch in which 15406 has been merged)

File: packages/core/strapi/lib/commands/utils/commander.js [hunk 1]
```diff
@@ -67,10 +68,21 @@ const promptEncryptionKey = async (thisCommand) => {
...
+ *
+ * @param {string} message The message to confirm with user
+ * @param {object} options Additional options
  */
 const confirmMessage = (message) => {
-  return async () => {
+  return async (command) => {
+    // if we have a force option, assume yes
+    const opts = command.opts();
+    if (opts?.force) {
```
Comment (Convly) [inline packages/core/strapi/lib/commands/utils/commander.js:80]: Do we want to ensure it's true or does a truth value is enough?

**Review by Convly — CHANGES_REQUESTED** [review]

File: packages/core/strapi/lib/commands/utils/commander.js (hunk 1 above)
Comment (innerdvations) [reply packages/core/strapi/lib/commands/utils/commander.js:80]: But I'll make --force a re-usable option as well, so nobody could accidentally implement it differently.

File: packages/core/strapi/lib/commands/utils/commander.js [hunk 2]
```diff
@@ -85,9 +97,15 @@ const confirmMessage = (message) => {
   };
 };
 
+const forceOption = new Option(
+  '-f, --force',
+  `Automatically nswer "yes" to all prompts, including potentially destructive requests, and run non-interactively.`
```
Comment (Convly) [inline packages/core/strapi/lib/commands/utils/commander.js:102]:
```suggestion
  `Automatically answer "yes" to all prompts, including potentially destructive requests, and run non-interactively.`
```

:angry:

**Review by Convly — CHANGES_REQUESTED** [review]

**Review by Convly — APPROVED** [review]
Front tests are failing but I don't think it's because of this PR.

**Pull request merged**